CRITICAL - 2022-03-28 17:39:32 --> Invalid file: welcome_message.php
#0 C:\xampp_8\htdocs\CI_BSI\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message...')
#1 C:\xampp_8\htdocs\CI_BSI\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#2 C:\xampp_8\htdocs\CI_BSI\app\Controllers\Home.php(9): view('welcome_message')
#3 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8\htdocs\CI_BSI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-28 17:42:02 --> Invalid file: welcome_message.php
#0 C:\xampp_8\htdocs\CI_BSI\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message...')
#1 C:\xampp_8\htdocs\CI_BSI\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#2 C:\xampp_8\htdocs\CI_BSI\app\Controllers\Home.php(9): view('welcome_message')
#3 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8\htdocs\CI_BSI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-28 17:42:23 --> Invalid file: welcome_message.php
#0 C:\xampp_8\htdocs\CI_BSI\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message...')
#1 C:\xampp_8\htdocs\CI_BSI\system\Common.php(1089): CodeIgniter\View\View->render('welcome_message', Array, true)
#2 C:\xampp_8\htdocs\CI_BSI\app\Controllers\Home.php(9): view('welcome_message')
#3 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8\htdocs\CI_BSI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-03-28 18:12:37 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-03-28 18:12:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp_8\htdocs\CI_BSI\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp_8\htdocs\CI_BSI\app\Models\model_home.php(15): CodeIgniter\Database\BaseConnection->query('SELECT * FROM a...')
#2 C:\xampp_8\htdocs\CI_BSI\app\Controllers\Home.php(10): App\Models\model_home->funcao_teste()
#3 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(832): App\Controllers\Home->index()
#4 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\xampp_8\htdocs\CI_BSI\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp_8\htdocs\CI_BSI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
